packages/fx-core/src/core/index.ts

```diff
@@ -549,6 +540,10 @@ class CoreImpl implements Core {
 
                     const subItem = await getSelectSubItem!(token, validFxProject);
                     this.ctx.treeProvider?.add([subItem[0]]);
+
+                    if (validFxProject && !subItem[1]) {
```

Comment:
Popup select sub if it's valid teamsfx project and user hasn't select one(don't popup if only one sub)